src/containers/tag/TagPosts.js

```diff
@@ -66,11 +66,13 @@ class TagPosts extends React.Component {
 }
 
 export default compose(
-  prefetch(({ dispatch, params: { tagName, id }, query }) =>
-    dispatch(fetchTag(tagName, id))
+  prefetch(({ dispatch, params: { tagName, id }, query }) => {
+    dispatch(resetNewPostCount(tagName, id))
```

Comment:
can you just do `if (id) dispatch(resetNewPostCount(tagname, id))` here and drop the `NO_ACTION` stuff in the action?